modules/configuring-logging-loki-ruler.adoc

```diff
@@ -30,14 +30,16 @@ metadata:
 spec:
 # ...
   rules:
-    enabled: true <1>
-    selector:
+    enabled: true #<1>
+    selector: #<2>
       matchLabels:
-        openshift.io/<label_name>: "true" <2>
-    namespaceSelector:
+        <label_name>: "true" #<3>
+    namespaceSelector: #<4>
       matchLabels:
-        openshift.io/<label_name>: "true" <3>
+        <label_name>: "true" #<5>
 ----
 <1> Enable Loki alerting and recording rules in your cluster.
-<2> Add a custom label that can be added to namespaces where you want to enable the use of logging alerts and metrics.
+<2> Specify the selector for the alerting and recording resources.
 <3> Add a custom label that can be added to namespaces where you want to enable the use of logging alerts and metrics.
+<4> Specify the namespaces in which the alerting and recording rules are defined for the {loki-op}. If undefined, only the rules defined in the same namespace as the `LokiStack` CR is are used.
```

Comment:
`CR is are used`

I would drop the "is" - but only a suggestion for later

Reply:
fixed